collector/iis.go

```diff
@@ -30,11 +29,11 @@ type simple_version struct {
 
 func getIISVersion() simple_version {
 	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `SOFTWARE\Microsoft\InetStp\`, registry.QUERY_VALUE)
+	defer k.Close()
```

Comment:
What is the reason for this change? In general, it might not be safe to call `Close()` if an error occurred, which is why you'd do it after the err-check. Was this causing some issue?